File: returned/python/today12/evaluate_reverse_polish_notation_test.py
from evaluate_reverse_polish_notation import Solution

def test1():
    assert Solution().evalRPN(["2","1","+","3","*"]) == 9
    
def test2():
    assert Solution().evalRPN(["4","13","5","/","+"]) == 6

def test3():
    assert Solution().evalRPN(["10","6","9","3","+","-11","*","/","*","17","+","5","+"]) == 22

File: csharp/LeetTests/Today1/EvaluateReversePolishNotationTest.cs
using Leet.EvaluateReversePolishNotation;

namespace LeetTests;

internal class EvaluateReversePolishNotationTest
{
    [Test]
    public void TestEvaluateReversePolishNotation()
    {
        var solution = new Solution();
        var result = solution.EvalRPN(["2", "1", "+", "3", "*"]);
        Assert.That(result, Is.EqualTo(9));
    }

    [Test]
    public void TestEvaluateReversePolishNotation2()
    {
        var solution = new Solution();
        var result = solution.EvalRPN(["4", "13", "5", "/", "+"]);
        Assert.That(result, Is.EqualTo(6));
    }

}